src/app/redux/Login/actions.js

```diff
@@ -1,40 +1,31 @@
 import UserService from '../../../services/UserService';
 
-export const actions = [
-  { GET_USERS: 'GET_USERS' },
-  { GET_USERS_SUCCESS: 'GET_USERS_SUCCESS' },
-  { GET_USERS_FAILURE: 'GET_USERS_FAILURE' },
-  { USER_LOGIN: 'USER_LOGIN' }
-];
+export const actions = {
+  USER_LOGIN_SUCCESS: 'USER_LOGIN_SUCCESS',
+  USER_LOGIN_FAILURE: 'USER_LOGIN_FAILURE'
+};
+
+const privateActionCreators = {
+  loginSuccess: (data, onSuccess) => {
+    onSuccess();
+    return {
+      type: actions.USER_LOGIN_SUCCESS,
+      payload: data
+    };
+  },
+  loginFailure: () => {
+    alert('Wrong email or password');
```

Comment:
receive this as a function, like `onSuccess()`, from the login `index`. Also move the text to `Login/strings.js`